src/core/meshcop/dtls.cpp

```diff
@@ -438,7 +440,16 @@ ThreadError Dtls::MapError(int rval)
 
 void Dtls::HandleMbedtlsDebug(void *ctx, int level, const char *file, int line, const char *str)
 {
-    otLogInfoMeshCoP("%s:%04d: %s\r\n", file, line, str);
+#ifdef WINDOWS_LOGGING
+
+    if (strnlen(str, 512) != 512)
+    {
```

Comment:
When I used the logs, the line and file number was just too much information (and a lot of text to save to file every line). The text from the logs was plenty enough to go and find the code it was referring too.